src/utils/commands.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+/* eslint-disable @typescript-eslint/restrict-template-expressions */
+import { resolve } from 'path';
+import Command from '@oclif/command';
+
+import { read, write, exists } from './fs';
+import { FILE_NAME } from '../constants';
+import { NetworkConfigLocal } from '../types';
+
+export const getCommandsToExecPostMigration = async (
+	networkConstant: NetworkConfigLocal,
+	outputDir: string,
+) => {
+	const commandsToExecute = [];
+
+	const forgingStatusJsonFilepath = resolve(outputDir, FILE_NAME.FORGING_STATUS);
+	if (await exists(forgingStatusJsonFilepath)) {
+		const forgingStatusString = (await read(forgingStatusJsonFilepath)) as string;
+		const forgingStatusJson = JSON.parse(forgingStatusString);
+
+		const chainID = parseInt(networkConstant.tokenID.substring(0, 2), 16);
+
+		for (const forgingStatus of forgingStatusJson) {
+			commandsToExecute.push(
+				'\n',
+				`## Generate/Register the BLS keys for validator ${forgingStatus.lskAddress} - Please modify the command if necessary`,
+				'\n',
+			);
+
+			const keysFilepath = resolve(outputDir, FILE_NAME.KEYS);
+			commandsToExecute.push(
+				`lisk-core keys:create --chainid ${chainID} --output ${keysFilepath} --add-legacy`,
+				`lisk-core keys:import --file-path ${keysFilepath}`,
+				`lisk-core endpoint:invoke random_setHashOnion '{ "address":"${forgingStatus.lskAddress}"}'`,
+				`lisk-core endpoint:invoke generator_setStatus '{ "address":"${forgingStatus.lskAddress}", "height": ${forgingStatus.height}, "maxHeightGenerated":  ${forgingStatus.maxHeightPreviouslyForged}, "maxHeightPrevoted":  ${forgingStatus.maxHeightPrevoted} }' --pretty`,
+				`lisk-core generator:enable ${forgingStatus.lskAddress} --use-status-value`,
+				'lisk-core transaction:create legacy registerKeys 4000000 --key-derivation-path=legacy --send',
```

Comment:
why fees `4000000`? maybe we can compute by bytes length? its not too much fee but users might tend to ask why so much fee paid